_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -87,7 +87,8 @@ VAProfile ConvertProfileTypeMFX2VAAPI(mfxU32 type)
 
 mfxU8 ConvertSliceStructureVAAPIToMFX(mfxU8 structure)
 {
-    switch (structure)
+    // Current implementation of Linux doesn't support SliceLevelRateCtrl
```

Comment:
The comment looks ambiguous. This function maps "VAConfigAttribEncSliceStructure" to "m_caps.SliceStructure".  It must always discard bit "VA_ENC_SLICE_STRUCTURE_MAX_SLICE_SIZE". Doesn't matter if max slice size (aka SliceLevelRateCtrl) supported by driver or not.